src/app/profile/payment-methods/payment-method/payment-method.component.js

```diff
@@ -0,0 +1,34 @@
+import angular from 'angular';
+import template from './payment-method.tpl';
+
+class PaymentMethodController{
+
+  /* @ngInject */
+  constructor(){
+    this.isCollapsed = true;
+  }
+
+  getExpiration() {
+    return `EXPIRES ${this.model.expiry_month}/${this.model.expiry_year}`;
```

Comment:
I think you can do this in the view. Doing translation is easier there. Was there a reason you did it here?